pom.xml

```diff
@@ -73,19 +72,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>docker-java-api</artifactId>
-            <version>3.2.13-37.vf3411c9828b9</version>
-        </dependency>
-
-        <dependency>
-            <groupId>com.github.docker-java</groupId>
-            <artifactId>docker-java-transport-httpclient5</artifactId>
-            <version>3.2.13</version>
+            <version>3.2.13-68.va_875df25a_b_45</version>
         </dependency>
```

Comment:
`docker-java-transport-httpclient5` is bundled in `docker-java-api` plugin as of [3.2.13-68.va_875df25a_b_45](https://updates.jenkins.io/download/plugins/docker-java-api/3.2.13-68.va_875df25a_b_45/docker-java-api.hpi).